src/button/README.md

```diff
@@ -2,7 +2,48 @@
 
 The `vscode-button` is a web component implementation of a [button element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button). The `vscode-button` also supports several visual appearances––primary, secondary, and icon.
 
-## Attributes
+![Button hero](/docs/assets/button-hero.png)
+
+## Usage
+
+### Types
+
+| Type      | Example                                               | Usage                                                                              |
+| --------- | ----------------------------------------------------- | ---------------------------------------------------------------------------------- |
+| Primary   | ![Primary](/docs/assets/button-types-primary.png)     | Emphasizes the highest priority action in a view.                                  |
+| Secondary | ![Secondary](/docs/assets/button-types-secondary.png) | Used for additional actions in a view that already features a primary action.      |
+| Icon      | ![Icon](/docs/assets/button-types-icon.png)           | A space-efficient style that renders a single icon to represent a specific action. |
+
+### Best Practices
+
+| ❌ Don't                                                        | ✅ Do                                                              |
+| --------------------------------------------------------------- | ------------------------------------------------------------------ |
+| ![Image placeholder](/docs/assets/img-placeholder.png)          | ![Image placeholder](/docs/assets/img-placeholder.png)             |
+| Don't use multiple primary buttons unless absolutely necessary. | Provide a single primary button with one or more secondary actions |
```

Comment:
I know this section is supposed to be succinct, but it would be great to have a little bit more insight on what "absolutely necessary" means or at least some rules of thumb since this could/will be very open to interpretation for the reader.

Reply:
Thought about this for a bit and decided to make it....even more succinct 😂

I think the biggest thing is not to use >1 primary buttons on the same component/right next to each other.